ocaml/idl/datamodel_cluster_host.ml

```diff
@@ -105,15 +105,31 @@ let t =
            ~default_value:(Some (VRef null_ref)) "Reference to the Host object"
        ; field ~qualifier:StaticRO ~lifecycle ~ty:Bool "enabled"
            ~default_value:(Some (VBool false))
-           "Whether the cluster host believes that clustering should be \
-            enabled on this host"
+           {|Whether the cluster host believes that clustering should be \
```

Comment:
This seems to break the SDK, somehow

Reply:
maybe it does not like quoted strings?

Reply:
removing the quoted string makes it happy again